DESCRIPTION

```diff
@@ -41,6 +41,7 @@ Depends:
     teal.slice (>= 0.5.1.9009)
 Imports:
     checkmate (>= 2.1.0),
+    crayon,
```

Comment:
@vedhav do you need to add this to pre-commit yaml?

Reply:
and verdepcheck part? later on in DESCRIPTION file?

Reply:
Oh yeah, I should start using the pre-commit.

Reply:
Yup my commits are 🐢now...
How long does it take, it's been over 3 mins for me now.

Reply:
Did not time it but it took over 4 mins for sure. Resolving it now.

Reply:
It takes more time during first build becuase it creates cache. If you have global R RENV CACHE it reuses your cache